### PR TITLE
MapFunctionLambdaExample fix

### PR DESCRIPTION
**Pull request by denvash**
For the current example, we need to map values to **upper case** (_and not to lower case_)

### CODE REVIEW

**Comment by denvash** [comment]
[clabot:check]

**Comment by guozhangwang** [comment]
@denvash could you elaborate why we need to change to upper case?

**Comment by mjsax** [comment]
@guozhangwang JavaDocs says that the example convert data to upper-case ;)

File: src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java [hunk 1]
```diff
@@ -120,7 +120,8 @@ public static void main(final String[] args) {
     final KStream<byte[], String> textLines = builder.stream("TextLinesTopic", Consumed.with(byteArraySerde, stringSerde));
 
     // Variant 1: using `mapValues`
-    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues(v -> v.toLowerCase());
+    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues(value -> value.toUpperCase());
+    //or use method reference: textLines.mapValues((ValueMapper<String, String>) String::toLowerCase);
```
Comment (mjsax) [inline src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java]: Comment should be updated to upper-case, too ;)

**Comment by denvash** [comment]
@mjsax Empty line for intending purpose, not necessary.  

screenshots as seen in `Intellij Documentation`
image

image

File: src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java [hunk 2]
```diff
@@ -50,7 +50,7 @@
  *
  * In this example, we count the TopN articles from a stream of page views (aka clickstreams) that
  * reads from a topic named "PageViews". We filter the PageViews stream so that we only consider
- * pages of type article, and then map the recored key to effectively nullify the user, such that
+ * pages of type article, and then map the recorded key to effectively nullify the user, such that
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java]: Typo: recorded key -> record key

File: src/main/java/io/confluent/examples/streams/WordCountLambdaExample.java [hunk 3]
```diff
@@ -70,6 +70,7 @@
  * results to the output topic "streams-wordcount-output".
  * <pre>
  * {@code
+ *
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/WordCountLambdaExample.java]: Newline not needed

File: src/main/java/io/confluent/examples/streams/ApplicationResetExample.java [hunk 4]
```diff
@@ -43,6 +43,7 @@
  * 2) Create the input, intermediate, and output topics used by this example.
  * <pre>
  * {@code
+ *
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/ApplicationResetExample.java]: I think this newline is not needed.

File: src/main/java/io/confluent/examples/streams/ApplicationResetExample.java [hunk 5]
```diff
@@ -58,13 +59,15 @@
  * Once packaged you can then run:
  * <pre>
  * {@code
- * $ java -cp target/kafka-streams-examples-4.0.0-SNAPSHOT-standalone.jar io.confluent.examples.streams.ApplicationResetExample
+ * $ java -cp target/kafka-streams-examples-4.1.0-standalone.jar \
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/ApplicationResetExample.java]: Thanks for paying attention to details such as this!

File: src/main/java/io/confluent/examples/streams/GlobalKTablesExample.java [hunk 6]
```diff
@@ -54,6 +54,7 @@
  * 2) Create the input/intermediate/output topics used by this example.
  * <pre>
  * {@code
+ *
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/GlobalKTablesExample.java]: Newline not needed

File: src/main/java/io/confluent/examples/streams/WordCountLambdaExample.java [hunk 7]
```diff
@@ -95,6 +96,7 @@
  * that interval will be compacted in memory.
  * <pre>
  * {@code
+ *
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/WordCountLambdaExample.java]: Newline not needed

File: src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java [hunk 8]
```diff
@@ -120,7 +120,8 @@ public static void main(final String[] args) {
     final KStream<byte[], String> textLines = builder.stream("TextLinesTopic", Consumed.with(byteArraySerde, stringSerde));
 
     // Variant 1: using `mapValues`
-    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues(v -> v.toLowerCase());
+    final KStream<byte[], String> uppercasedWithMapValues = textLines.mapValues(value -> value.toUpperCase());
```
Comment (miguno) [inline src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java]: Thanks for catching this (lower- > upper).  Not sure how this slipped in. :-)

**Comment by miguno** [comment]
Thanks for this PR @denvash !

**Comment by mjsax** [comment]
@denvash I jut realized that we never merged this PR -- sorry that it slipped through. If you could rebase the PR, I can merge it (if you are still interested).

**Comment by denvash** [comment]
> @denvash I jut realized that we never merged this PR -- sorry that it slipped through. If you could rebase the PR, I can merge it (if you are still interested).

Sure! Check #194 (Sry I couldn't figure out how to rebase in same thread)

**Comment by mjsax** [comment]
Thanks. Closing this a we have a new PR.

**Pull request closed without merging**